Formula/cquery.rb

```diff
@@ -13,10 +13,15 @@ class Cquery < Formula
     sha256 "9692d28682c009a9bec583ff20aa16ce5c790975375755a5a48a28ec3e3953c4" => :el_capitan
   end
 
+  depends_on :macos => :sierra
+  depends_on "cmake" => :build
+  depends_on "llvm"
```

Comment:
What's the upside of using our `llvm` over the bundled one you mentioned here? Share some concern about making future `llvm` updates potentially harder to get through in a reasonable timescale.